src/components/PastTrips/PastTripsList.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import trips from './trips.json';
+import PastTripsCard from './PastTripsCard';
+import addTrip from './addTrip.svg';
+
+const PastTripsList = () => {
+  const [tripList] = useState(trips);
+
+  console.log(tripList);
```

Comment:
Let's remove this console.log statement. Our linting step will take care of these (or at least flag them automatically soon.